experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveTableView.java

```diff
@@ -65,4 +69,14 @@ public String getBaseLocation() {
   public List<FieldSchema> getPartitionCols() {
     return mPartitionCols;
   }
+
+  @Override
+  public TableViewInfo toProto(String viewName) {
+    // TODO(gpang): implement clustered columns.
+    return TableViewInfo.newBuilder()
+        .setViewName(viewName)
+        .setViewType("todo: hive parquet")
+        .addClusteredColumns(mTable.getSchema().getCols(mTable.getSchema().getColsCount() - 1))
+        .build();
+  }
```

Comment:
i think the information you are looking for here is stored in `mPartitionCols`, the list of columns the table is partitioned / clustered by?

Reply:
wow, I didn't realize that. Updated.